library/error.c

```diff
@@ -110,6 +102,10 @@
 #include "mbedtls/error.h"
 #endif
 
+#if defined(MBEDTLS_PLATFORM_C)
+#include "mbedtls/platform.h"
```

Comment:
Same piece of code was removed at lines 165~168, is this intentional?

Reply:
Most probably yes, as the error.c is auto-generated by script.

Reply:
oh, alright :)